frontend/src/test/java/com/google/tripmeout/serialization/GsonPlaceVisitTypeAdapterTest.java

```diff
@@ -55,8 +55,14 @@ public void deserialize_noPlaceId_throwsJsonParseException() throws Exception {
   }
   @Test
   public void deserialize_noName_throwsJsonParseException() throws Exception {
-    assertThrows(JsonParseException.class,
-        () -> gson.fromJson(TestDataAccessUtil.getPlaceVisitWithoutName(), PlaceVisitModel.class));
+    PlaceVisitModel place = gson.fromJson(
+        TestDataAccessUtil.getPlaceVisitWithoutName(), PlaceVisitModel.class);
+    assertThat(place.placeId()).isEqualTo("ABC");
```

Comment:
This would be much more readable as a single `assertThat(place).isEqualTo(expectedPlace)`, where `expectedPlace` is a `PlaceVisitModel`. AutoValue implements a correct `equals` method for you.